corehq/apps/reports/views.py

```diff
@@ -1756,7 +1756,14 @@ def _error(msg):
         instance = _get_location_safe_form(domain, request.couch_user, instance_id)
         context = _get_form_context(request, domain, instance)
         if not instance.app_id or not instance.build_id:
-            return _error(_('Could not detect the application/form for this submission.'))
+            deviceID = instance.metadata.deviceID
+            if deviceID and deviceID == 'Formplayer':
```

Comment:
unfortunate not :/ https://github.com/dimagi/formplayer/blob/f47d3bf7a749f4ee13ad2e92745906794ae0088f/src/main/java/session/FormplayerInstanceInitializer.java#L79